app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/NoCollectionsMessageViewHolder.kt

```diff
@@ -5,59 +5,61 @@
 package org.mozilla.fenix.home.sessioncontrol.viewholders
 
 import android.view.View
-import androidx.core.view.isVisible
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.unit.dp
 import androidx.lifecycle.LifecycleOwner
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.flow.collect
-import kotlinx.coroutines.flow.map
 import mozilla.components.browser.state.selector.normalTabs
 import mozilla.components.browser.state.store.BrowserStore
-import mozilla.components.lib.state.ext.flowScoped
-import mozilla.components.support.ktx.kotlinx.coroutines.flow.ifChanged
+import mozilla.components.lib.state.ext.observeAsComposableState
 import org.mozilla.fenix.R
-import org.mozilla.fenix.databinding.NoCollectionsMessageBinding
-import org.mozilla.fenix.ext.increaseTapArea
+import org.mozilla.fenix.compose.CollectionsPlaceholder
+import org.mozilla.fenix.compose.ComposeViewHolder
 import org.mozilla.fenix.home.sessioncontrol.CollectionInteractor
-import org.mozilla.fenix.utils.view.ViewHolder
 
-@OptIn(ExperimentalCoroutinesApi::class)
-open class NoCollectionsMessageViewHolder(
-    view: View,
+/**
+ * [RecyclerView.ComposeViewHolder] for displaying a message detailing the collections feature and
+ * allowing users to easily start creating their first.
+ *
+ * @param composeView [ComposeView] which will be populated with Jetpack Compose UI content.
+ * @param viewLifecycleOwner [LifecycleOwner] to which this Composable will be tied to.
+ * @param store to receive live updates to normal tabs.
+ * @param interactor [CollectionInteractor] callback for user interaction.
+ */
+class NoCollectionsMessageViewHolder(
+    composeView: ComposeView,
     viewLifecycleOwner: LifecycleOwner,
-    store: BrowserStore,
-    interactor: CollectionInteractor
-) : ViewHolder(view) {
+    private val store: BrowserStore,
+    private val interactor: CollectionInteractor
+) : ComposeViewHolder(composeView, viewLifecycleOwner) {
 
     init {
-        val binding = NoCollectionsMessageBinding.bind(view)
+        val horizontalPadding =
+            composeView.resources.getDimensionPixelSize(R.dimen.home_item_horizontal_margin)
+        composeView.setPadding(horizontalPadding, 0, horizontalPadding, 0)
+    }
 
-        binding.addTabsToCollectionsButton.apply {
+    companion object {
+        val LAYOUT_ID = View.generateViewId()
+    }
 
-            setOnClickListener {
-                interactor.onAddTabsToCollectionTapped()
-            }
-            isVisible = store.state.normalTabs.isNotEmpty()
-        }
+    @Composable
+    override fun Content() {
+        val normalTabsState = store.observeAsComposableState { state -> state.normalTabs }.value ?: emptyList()
+        Column {
```

Comment:
```suggestion
        val normalTabsState = store.observeAsComposableState { state -> state.normalTabs }.value ?: emptyList()
        
        Column {
```
New line to separate out the component from the variables